File: An2/Sem1/MAP/MAP-C#/Partial/Partial/Repository/RepoInterface.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Partial.Repository
{
    interface RepoInterface<T>
    {
        LinkedList<T> getElements();
        void writeToFile(LinkedList<T> elementsToWrite);
        void replaceContent(LinkedList<T> content);
    }
}

File: An2/Sem1/MAP/MAP-Java/Exam/src/Repository/RepoInterface.java
package Repository;

import java.util.ArrayList;

/**
 * Created by mihai on 2/3/14.
 */
public interface RepoInterface<T> {

    public ArrayList<T> getElements();
    public void writeToFile(ArrayList<T> elementsToWrite);
    public void replaceContent(ArrayList<T> content);
    public void addElement(T element);

}
